website/components/plugin-tier-label/style.module.css

```diff
@@ -27,6 +27,12 @@ to match Terraform Registry tier labels.
   }
 }
 
+/* add margin-top to separate from the adjacent
+search bar present on docs pages */
+:global(.g-search) + .root {
```

Comment:
I don't love the `:global` selector here, but I'm not sure how else to approach this. One option might be to make this CSS change within `docs-page`? Open to any and all thoughts here.